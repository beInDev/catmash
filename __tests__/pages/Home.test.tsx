import React from 'react'
import { render } from '@testing-library/react'
import Home from '../../pages/index'
import renderer from 'react-test-renderer'


test('renders deploy link', () => {
  const { getByText } = render(<Home />)
  const linkElement = getByText(
    /Instantly deploy your Next\.js site to a public URL with Vercel\./
  )
  expect(linkElement).toBeInTheDocument()
})

it('renders homepage unchanged', () => {
  const tree = renderer.create(<Home />).toJSON()
  expect(tree).toMatchSnapshot()
})