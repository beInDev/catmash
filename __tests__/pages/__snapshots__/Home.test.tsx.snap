// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-justify-xs-center"
>
  <div
    className="MuiGrid-root MuiGrid-item"
  >
    <div
      className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
    >
      <button
        className="MuiButtonBase-root MuiCardActionArea-root"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <div
          className="MuiCardContent-root makeStyles-card-1"
        >
          <div
            className="MuiCardMedia-root makeStyles-media-2"
            style={
              Object {
                "backgroundImage": "url(\\"http://google.fr\\")",
              }
            }
            title="cat :3"
          />
        </div>
        <span
          className="MuiCardActionArea-focusHighlight"
        />
      </button>
    </div>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item"
  >
    <div
      className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
    >
      <button
        className="MuiButtonBase-root MuiCardActionArea-root"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <div
          className="MuiCardContent-root makeStyles-card-1"
        >
          <div
            className="MuiCardMedia-root makeStyles-media-2"
            style={
              Object {
                "backgroundImage": "url(\\"http://google.fr\\")",
              }
            }
            title="cat :3"
          />
        </div>
        <span
          className="MuiCardActionArea-focusHighlight"
        />
      </button>
    </div>
  </div>
</div>
`;
